app/assets/javascripts/discourse/app/templates/modal/create-account.hbs

```diff
@@ -1,172 +1,152 @@
 {{#create-account email=accountEmail disabled=submitDisabled action=(action "createAccount")}}
   {{#unless complete}}
     {{plugin-outlet name="create-account-before-modal-body"}}
-    {{#d-modal-body title="create_account.title" class=modalBodyClasses}}
-
-      {{#unless hasAuthOptions}}
-        {{login-buttons externalLogin=(action "externalLogin")}}
-      {{/unless}}
+    {{#d-modal-body class=modalBodyClasses}}
 
       {{#if skipConfirmation}}
         {{loading-spinner size="large"}}
       {{/if}}
 
+      {{#unless hasAuthOptions}}
+        <div class="create-account-login-buttons">
+          {{login-buttons externalLogin=(action "externalLogin")}}
+        </div>
+      {{/unless}}
+
       {{#if showCreateForm}}
-        <div class="login-form">
-          <form>
-            <table>
-              <tbody>
-                <tr class="input create-account-email">
-                  <td class="label">
-                    <label for="new-account-email">
-                      {{i18n "user.email.title"}}
-                      {{~#if userFields~}}
-                        <span class="required">*</span>
-                      {{/if}}
-                    </label>
-                  </td>
-                  <td>
-                    {{#if emailValidated}}
-                      <span class="value">{{accountEmail}}</span>
-                    {{else}}
-                      {{input type="email" value=accountEmail id="new-account-email" name="email" autofocus="autofocus"}}
-                    {{/if}}
-                  </td>
-                </tr>
-
-                <tr class="instructions create-account-email">
-                  <td></td>
-                  {{input-tip validation=emailValidation id="account-email-validation"}}
-                  <td><label>{{i18n "user.email.instructions"}}</label></td>
-                </tr>
-
-                <tr class="input">
-                  <td class="label">
-                    <label for="new-account-username">
-                      {{i18n "user.username.title"}}
-                      {{~#if userFields~}}
-                        <span class="required">*</span>
-                      {{/if}}
-                    </label>
-                  </td>
-                  <td>
-                    {{#if usernameDisabled}}
-                      <span class="value">{{accountUsername}}</span>
-                    {{else}}
-                      {{input value=accountUsername id="new-account-username" name="username" maxlength=maxUsernameLength autocomplete="discourse"}}
+        <div class="create-account-form">
+          <div class="login-welcome-header">
+            <h1>Welcome!</h1> <img src="/images/emoji/twitter/wave/{{skinToneId}}.png" alt="" class="waving-hand">
```

Comment:
welcome is not translated?

Reply:
The image URL won't work with subfolder and the `emoji_set` site setting be used instead of hardcoding `twitter`?